jobs/cloud engineer_crowdstrike

```diff
@@ -1,88 +1,82 @@
----
-title:Senior Cloud Engineer with Python
-company: CrowdStrike
-url: https://www.crowdstrike.com/careers/?p=job%2FoQ3F6fwo
-location: Bucharest,Romania
-# Choose one of the following options
-contract: permanent 
-contact:
-    name: Ana Robu
-    email: ana.robu@crowdstrike.com
+--
+		+title:Senior Cloud Engineer with Python
```

Comment:
Hi, it looks like you've indented every line of the file. That's completely unnecessary and will break the compiler!

Can you refer to the example text on our readme page?
https://github.com/pythonjobs/jobs

That shows you exactly how to format a job ad.